modules/lwjgl/openxr/src/generated/java/org/lwjgl/openxr/XR10.java

```diff
@@ -0,0 +1,1577 @@
+        }
+        return nxrGetInputSourceLocalizedName(session, getInfo.address(), remainingSafe(buffer), memAddress(bufferCountOutput), memAddressSafe(buffer));
+    }
+
+    // --- [ xrApplyHapticFeedback ] ---
+
+    public static int nxrApplyHapticFeedback(XrSession session, long hapticActionInfo, long hapticFeedback) {
+        long __functionAddress = session.getCapabilities().xrApplyHapticFeedback;
+        if (CHECKS) {
+            XrHapticActionInfo.validate(hapticActionInfo);
+        }
+        return callPPPI(session.address(), hapticActionInfo, hapticFeedback, __functionAddress);
+    }
+
+    @NativeType("XrResult")
+    public static int xrApplyHapticFeedback(XrSession session, @NativeType("XrHapticActionInfo const *") XrHapticActionInfo hapticActionInfo, @NativeType("XrHapticBaseHeader const *") XrHapticBaseHeader hapticFeedback) {
+        return nxrApplyHapticFeedback(session, hapticActionInfo.address(), hapticFeedback.address());
+    }
+
+    // --- [ xrStopHapticFeedback ] ---
+
+    public static int nxrStopHapticFeedback(XrSession session, long hapticActionInfo) {
+        long __functionAddress = session.getCapabilities().xrStopHapticFeedback;
+        if (CHECKS) {
+            XrHapticActionInfo.validate(hapticActionInfo);
+        }
+        return callPPI(session.address(), hapticActionInfo, __functionAddress);
+    }
+
+    @NativeType("XrResult")
+    public static int xrStopHapticFeedback(XrSession session, @NativeType("XrHapticActionInfo const *") XrHapticActionInfo hapticActionInfo) {
+        return nxrStopHapticFeedback(session, hapticActionInfo.address());
+    }
+
+    // --- [ XR_MAKE_VERSION ] ---
+
+    /**
+     * Constructs an API version number.
+     * 
+     * <p>This macro <b>can</b> be used when constructing the {@link XrApplicationInfo}{@code ::pname:apiVersion} parameter passed to {@link #xrCreateInstance CreateInstance}.</p>
+     *
+     * @param major the major version number
+     * @param minor the minor version number
+     * @param patch the patch version number
+     */
+    @NativeType("uint64_t")
+    public static long XR_MAKE_VERSION(@NativeType("uint32_t") int major, @NativeType("uint32_t") int minor, @NativeType("uint32_t") int patch) {
+        return ((major & 0xffffL) << 48) | ((minor & 0xffffL) << 32) | patch;
+    }
+
+    // --- [ XR_VERSION_MAJOR ] ---
+
+    /**
+     * Extracts the API major version number from a packed version number.
+     *
+     * @param version the OpenXR API version
+     */
+    @NativeType("uint64_t")
+    public static long XR_VERSION_MAJOR(@NativeType("uint64_t") long version) {
+        return (version >>> 48) & 0xffffL;
+    }
+
+    // --- [ VK_VERSION_MINOR ] ---
+
+    /**
+     * Extracts the API minor version number from a packed version number.
+     *
+     * @param version the OpenXR API version
+     */
+    @NativeType("uint64_t")
+    public static long VK_VERSION_MINOR(@NativeType("uint64_t") long version) {
```

Comment:
I think this should be XR_VERSION_MINOR instead of VK_VERSION_MINOR (and the same holds for VK_VERSION_PATCH)

Reply:
Noted, I'll rename these once I have more time on my hands